terraform/modules/logsearch/elb_platform_kibana.tf

```diff
@@ -0,0 +1,25 @@
+resource "aws_elb" "platform_kibana_elb" {
+  name = "${var.stack_description}-platform-kibana"
+  subnets = ["${var.elb_subnets}"]
+  security_groups = ["${var.restricted_security_group}"]
+  idle_timeout = 3600
+
+  listener {
+    instance_port = 80
```

Comment:
I get this is in the restricted web traffic security group, but still should be HTTPS with `*` cert, not plain HTTP, yeah?